src/opentelemetry.ts

```diff
@@ -33,3 +33,16 @@ export const opentelemetrySDK = new NodeSDK({
   instrumentations: [...getNodeAutoInstrumentations()]
 });
 opentelemetrySDK.start();
+
+process.on('beforeExit', async () => {
```

Comment:
We're using terminus to ensure shutdown is made on INT signal. Hooking onto `beforeExit` and awaiting shutdown can override that ,which is not what we'd want. However, I think we can live with it in this instance given the import limitations on this module.

Reply:
We can't hook to terminus as this script runs before the "application code." Using the `beforeExit` event is just fine because it runs immediately before the termination.

Reply:
Problem is that it can stop the termination from happening if sdk shutdown hangs, that is after terminus calls exit, correct?

It is not a blocker as stated above, solution would likely be quite hacky.

Reply:
Yes, this can happen, but should not, I can add an timeout.

Reply:
Good idea, something of sorts  `await Promise.race([shutdown(), setTimeout(5000)]`